apps/canary-atomy-api/src/Service/Tenant/Adapters/AdminCreatorAdapter.php

```
@@ -0,0 +1,27 @@
+namespace App\Service\Tenant\Adapters;
+
+use App\Repository\UserRepository;
+use Nexus\TenantOperations\Contracts\AdminCreatorAdapterInterface;
+
+final readonly class AdminCreatorAdapter implements AdminCreatorAdapterInterface
+{
+    public function __construct(
+        private UserRepository $userRepository
+    ) {}
+
+    public function create(string $tenantId, string $email, string $password, bool $isAdmin = false): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$password'. (undefined)

(UnusedFormalParameter)